enterprise/messages.py

```diff
@@ -13,34 +14,108 @@
     configuration_helpers = None
 
 
-def add_consent_declined_message(request, enterprise_customer, course_run):
+def add_consent_declined_message(request, enterprise_customer, item):
     """
     Add a message to the Django messages store indicating that the user has declined data sharing consent.
 
     Arguments:
         request (HttpRequest): The current request.
         enterprise_customer (EnterpriseCustomer): The EnterpriseCustomer associated with this request.
-        course_run (dict): A dictionary containing information about the course run the user had declined consent for.
+        item (str): A string containing information about the item for which consent was declined.
     """
     messages.warning(
         request,
         _(
-            '{strong_start}We could not enroll you in {em_start}{course_title}{em_end}.{strong_end} '
+            '{strong_start}We could not enroll you in {em_start}{item}{em_end}.{strong_end} '
             '{span_start}If you have questions or concerns about sharing your data, please contact your learning '
             'manager at {enterprise_customer_name}, or contact {link_start}{platform_name} support{link_end}.{span_end}'
         ).format(
-            course_title=course_run['title'],
+            item=item,
             em_start='<em>',
             em_end='</em>',
             enterprise_customer_name=enterprise_customer.name,
             link_start='<a href="{support_link}" target="_blank">'.format(
-                support_link=configuration_helpers.get_value('ENTERPRISE_SUPPORT_URL')
+                support_link=configuration_helpers.get_value(
+                    'ENTERPRISE_SUPPORT_URL',
+                    getattr(settings, 'ENTERPRISE_SUPPORT_URL')  # Remove the `getattr` when setting is upstreamed.
```

Comment:
This all looks right, except you could provide a default value of `''` or `None` to `getattr`until we have things upstreamed so we don't get the 500 error.

Reply:
I'll do that before I log so we stop getting errors and update the sandbox.